### PR TITLE
Add info about the project to the home page

### PR DESCRIPTION
**Pull request by isaacdurazo**

### CODE REVIEW

**Comment by rwaldron** [comment]
I resolved the conflict and ran the linter with --fix

File: client/components/Home/index.jsx [hunk 1]
```diff
@@ -1,8 +1,44 @@
...
+                    presenting test results to AT vendors. The tests are based
+                    on examples from WAI-ARIA Authoring Practices, and are
+                    vetted with stakeholders following the working mode process.
+                </p>
+
+                <h2>Get involved</h2>
+                <ul>
+                    <li>
+                        <a href="https://www.w3.org/community/aria-at/">
+                            Join the community group
```
Comment (boazsender) [inline client/components/Home/index.jsx:21]: how?

File: client/components/Home/index.jsx [hunk 2]
```diff
@@ -1,8 +1,44 @@
...
+                        </a>
+                    </li>
+                    <li>
+                        <a>Sign up as a tester</a>
```
Comment (boazsender) [inline client/components/Home/index.jsx:25]: how?

File: client/components/Home/index.jsx [hunk 3]
```diff
@@ -1,8 +1,44 @@
...
+                    </li>
+                    <li>
+                        <a>Write more tests</a>
```
Comment (boazsender) [inline client/components/Home/index.jsx:28]: link to documentation.

Comment (boazsender) [reply client/components/Home/index.jsx:28]: https://github.com/w3c/aria-at/wiki/How-to-contribute-tests

File: client/components/Home/index.jsx [hunk 4]
```diff
@@ -1,8 +1,44 @@
...
+                    </li>
+                    <li>
+                        <a>Review the assertions of existing test plans</a>
+                    </li>
+                    <li>
+                        <a>Review test results</a>
+                    </li>
+                    <li>
+                        <a>Fix a good first issue</a>
```
Comment (boazsender) [inline client/components/Home/index.jsx:37]: https://github.com/w3c/aria-at/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by zcorpan** [comment]
Thank you @isaacdurazo @rwaldron @boazsender  !

I've addressed your comments, Boaz.

**Pull request merged**